src/main/webapp/blur-helpers.js

```diff
@@ -259,3 +273,122 @@ async function loadBlurredPhoto(photo) {
   };
   return getImageWithBlurredAreas(imageDetails, blurRadius);
 }
+
+/**
+ * Function that preloads some of the photos in sessionStorage.
+ * Assigns each photo a value and uses the Knapsack algorithm to decide which
+ * photos to load in maximum CACHE_SIZE space so that the sum of their value is
+ * the biggest.
+*/
+async function preloadPhotos() {
+  // Fetch photos information from the server.
+  const photosResponse = await fetch('/photos');
+  const photos = await photosResponse.json();
+
+  if (photos.length === 0) {
+    return;
+  }
+
+  // Assign each photo a value.
+  const serverTimeResponse = await fetch('server-time');
+  const currentDate = Date.parse(await serverTimeResponse.text());
```

Comment:
You may want to indicate that it holds the value is milliseconds.

Reply:
Added a comment for it.